charts/bulk-scan-processor/values.preview.template.yaml

```diff
@@ -37,6 +37,7 @@ java:
     PROCESS_PAYMENTS_ENABLED: "false"
     OCR_VALIDATION_URL_BULKSCAN_SAMPLE_APP: "http://bulk-scan-sample-app-aat.service.core-compute-aat.internal"
     DELETE_REJECTED_FILES_ENABLED: "false"
+    DELETE_COMPLETE_FILES_ENABLED: "false"
```

Comment:
if we are running functional tests then we will need this enabled else functional test will fail